test/test_namedtensor.py

```diff
@@ -544,6 +555,13 @@ def test_split_fns_propagates_names(self):
                 for split in splits:
                     self.assertEqual(split.names, orig_tensor.names)
 
+    def test_any_all(self):
+        for device in torch.testing.get_all_device_types():
+            x = torch.zeros(3, dtype=torch.bool, device=device, names=('C',))
+            # The result can't have any names so this is just a smoke test
```

Comment:
should we explicitly assert that dim names is empty?

Reply:
It's impossible for a scalar tensor to have any names: there is an invariant that a tensor must have number of names equal to number of dimensions. I think asserting it here would be redundant because other tests also make this assumption.

Reply:
right, but we could test where we reduce a multi-dim tensor along a single dimension.

Reply:
That's a good point, those tests exist. I've tossed the check in that the names are empty.